plugins/source/homebrew/.gitignore

```diff
@@ -1,3 +1,3 @@
 test
 dist
-./homebrew
\ No newline at end of file
+/homebrew
```

Comment:
```suggestion
homebrew
```

Reply:
No, there is an internal directory called `homebrew` that I don't want to ignore